### ISSUE TITLE
news - image with no accessible name

### ISSUE THREAD

**Issue by IdaAkiwumi** [chnm/rrchnm2016#100]
This image is being flagged by ANDI as having no accessible name.

![Screen Shot 2020-10-06 at 3 30 44 PM](https://user-images.githubusercontent.com/618307/95251056-fb9b5400-07e8-11eb-9d42-3a5bc726546b.png)

**Comment by kimisgold** [comment]
I just updated the alt text within the Wordpress dashboard.
